README.md

```diff
@@ -1,15 +1,77 @@
-# StuPro TOSCAna preliminary design study
+# TOSCA2vSphere
 [![Build Status](https://travis-ci.org/nfode/stupro_toscana_vorprojekt.svg?branch=master)](https://travis-ci.org/nfode/stupro_toscana_vorprojekt) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/46bbcc976f084a19a01300393554adf0)](https://www.codacy.com/app/nfode/stupro_toscana_vorprojekt?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=nfode/stupro_toscana_vorprojekt&amp;utm_campaign=Badge_Grade)
 [![codecov](https://codecov.io/gh/nfode/stupro_toscana_vorprojekt/branch/master/graph/badge.svg)](https://codecov.io/gh/nfode/stupro_toscana_vorprojekt)
 
+This is the repository for the preliminary design study for the study project TOSCAna. The project can be split into two parts:
```

Comment:
I would replace:
> The project can be split into two parts:

with:

> The project consists of the following two subprojects:

Subprojects might also not be the best wording here. Parts could also be acceptable, however *can* is definately not properly chosen.